### DISCUSSION TITLE
Last Digit in Input Mask Hides if Pattern Is Not "d.m.Y" in Vueform

### DISCUSSION THREAD

**Discussion by jajedidiah** [vueform/vueform#374] [Questions] [answered]
### Environment

Vueform version: 1.11.0
Vueform Mask version: 1.0.7
Vue version: 3.4.37
Theme: Tailwind
Browser: Chrome
Package Manager: npm
OS: MacOS Sequoia 15.1

### Reproduction

1. Configure a Vueform input field with a date mask pattern other than "d.m.Y", e.g., "dd.mm.YYYY".
2. Type a date value, ensuring it matches the mask format.
3. Observe that the last digit of the date is hidden in the input field.

### Sandbox

https://stackblitz.com/edit/github-ukstaz?file=src%2FApp.vue,vueform.config.js

### Describe the bug

While attempting to create a TextElement date mask in Vueform that accepts the mm/dd/yyyy format, the last digit of the date is hidden unless the mask pattern is specifically set to "d.m.Y". Using any other pattern, such as mm/dd/yyyy, causes the input field to hide the last digit, making the full date unreadable to the user.

### Additional context

- This behavior occurs only with custom date formats. The default "d.m.Y" pattern displays the entire date correctly.
- The issue persists across different separators or variations in the date format.

### Logs

```shell-script
None
```

**Comment by adamberecz** [comment, marked as answer]
The `format` must return the string in the format of `pattern` which must also match what `parse` expects. Here's a working version: https://stackblitz.com/edit/github-ukstaz-xmczl5?file=src%2FApp.vue,vueform.config.js
